### ISSUE TITLE
EFDesigner 2.0.1 won't create entities in the efmodel for existing poco .cs files

### ISSUE THREAD

**Issue by IoTFier** [msawczyn/EFDesigner#182]
Hello @msawczyn, good morning !

Attached repro solution has a sqlite3 database as shown in by the dgml file (pic attached below) inside path Test.EFDesigner.Data\Documents. 
My dev environment details - Win 10 64 v1909 | Visual Studio 16.5.4

I generated database table classes for this sqlite3 db using command "dotnet ef dbcontext scaffold" - output location @ Test.EFDesigner.Domain\Entities.

When I tried to drag-drop these classes on a efmodel diagram - Test.EFDesigner.EFVisualEditor\Documents\chinook_noaffinity.efmodel - 
1. **I don't see entities created in the diagram panel.** It stays blank. You will find generated classes only when you get in the properties of the efmodel file. This was working just fine in https://github.com/msawczyn/EFDesigner/releases/tag/v1.3.0.12

2. **When I tried to save the diagram, it shows errors as shown in the screenshot below** - again this was not the issue in https://github.com/msawczyn/EFDesigner/releases/tag/v1.3.0.12

3. For tables like media_types, artist, genres - should the EFDesigner be creating Enums ?

Am I missing any configuration or setting that will address above issues ?

**Repro Solution -**
[Test.EFDesigner.zip](https://github.com/msawczyn/EFDesigner/files/4501458/Test.EFDesigner.zip)

**Errors shown when tried to save .efmodel** 
![image](https://user-images.githubusercontent.com/26844611/79718081-d2b9ed80-82a0-11ea-8e40-c8e7fc2d8f69.png)

**dgml of the sqlite3 database that shows ER relationship (primary and foreign keys)** 
![image](https://user-images.githubusercontent.com/26844611/79717727-0e07ec80-82a0-11ea-8038-8da2e0381644.png)

**Comment by msawczyn** [comment]
Thanks for the report.

To test, I used the v2.0.1 pre-release designer in VStudio 2019 v16.5.3. I selected all the 11 files in `Test.EFDesigner.Domain/Entities` and dragged them onto a new design surface that I created in `Test.EFDesigner.EFVisualEditor`. I also dragged them from the file explorer onto a new design surface. 

_1. I don't see entities created in the diagram panel. It stays blank. You will find generated classes only when you get in the properties of the efmodel file. This was working just fine in https://github.com/msawczyn/EFDesigner/releases/tag/v1.3.0.12_
Can't reproduce. The design surface populates for me in both drag/drop cases.

_2. When I tried to save the diagram, it shows errors as shown in the screenshot below - again this was not the issue in https://github.com/msawczyn/EFDesigner/releases/tag/v1.3.0.12_

The screenshot shows two validation error types. The first, indicating that a class doesn't have an identifier defined, is valid. None of the classes generated by `dotnet ef dbcontext scaffold` indicated which property was the identity. The modeler can't make assumptions based on naming conventions - if it did, it would be wrong the majority of the time. 

The second is a bug, but it's a bug that's been there since at least December, if the git history is any clue. The association roles wern't getting set automatically for bidirectional associations. That error message should only be seen when the cardinalities mentioned are present, and that's clearly not the case with the associations in the model. That's fixed now and is available in the [2.0.2 Prerelease](https://github.com/msawczyn/EFDesigner/releases/tag/v2.0.2) release.

_3. For tables like media_types, artist, genres - should the EFDesigner be creating Enums ?_

No. Your input was a class, so the designer created a class.
